src/pages/billing/Project.js

```diff
@@ -233,6 +262,23 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
       const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
       const costFormatter = new Intl.NumberFormat(navigator.language, { style: 'currency', currency: spend.spendSummary.currency })
       setTotalCost(costFormatter.format(spend.spendSummary.cost))
+
+      const workspaceDetails = _.find(details => details.aggregationKey === 'Workspace')(spend.spendDetails)
+      console.assert(workspaceDetails !== undefined, 'Spend report details do not include aggregation by Workspace')
+      // Get the most expensive workspaces, sorted from most to least expensive.
+      const mostExpensiveWorkspaces = _.flow(
+        _.sortBy(({ cost }) => { return parseFloat(cost) }),
```

Comment:
🤔 We should ticket up API-side sorting + limiting for the spend detail array if we haven't already.

Reply:
I guess… I can certainly see sorting on the server side, but I'm not sure about limiting. If we limit, I feel like there has to be a parameter to specify the limit.

I think ultimately we are going to allow users to select the Workspaces they want to see, and in that case I am going to need all the Workspaces anyway, and I will filter what I expose to the chart.